### ISSUE TITLE
When Attempting HTH is Possible

### ISSUE THREAD

**Issue by ricegf** [ricegf/elven-fire#8]
The rules should state that HTH can only be attempted if a melee attack is otherwise permissible.

**Comment by sarinilla** [comment]
When is a melee attack impermissible?  We are not precluding the "run up and pounce" strategy, are we?

**Comment by ricegf** [comment]
No, not nerfing "pounce". Need to look a bit more at a clear definition.  _sigh_
